### ISSUE TITLE
`Lattice.dump` error:  'sudachipy.latticenode.LatticeNode' object has no attribute 'is_defined'

### ISSUE THREAD

**Issue by adrianeboyd** [WorksApplications/SudachiPy#128]
When logging is enabled, I noticed a bug in `Lattice.dump`, which might be related to the recent cython-related modifications? The error looks like this, calling `sudachipy` from `spacy`:

```
  File "/tmp/venv/lib/python3.7/site-packages/spacy/lang/ja/__init__.py", line 116, in get_dtokens
    tokens = tokenizer.tokenize(text)
  File "sudachipy/tokenizer.pyx", line 149, in sudachipy.tokenizer.Tokenizer.tokenize
  File "sudachipy/lattice.pyx", line 134, in sudachipy.lattice.Lattice.dump
AttributeError: 'sudachipy.latticenode.LatticeNode' object has no attribute 'is_defined'
```

**Comment by sorami** [comment]
@adrianeboyd Thank you for the notice, let me have a look.

**Comment by sorami** [comment]
I confirmed that bug does not happen with v0.4.5 which is pre-Cython.

**Comment by sorami** [comment]
The fix released in [v0.4.8](https://github.com/WorksApplications/SudachiPy/releases/tag/v0.4.8)

**Comment by adrianeboyd** [comment]
Thanks for the quick fix!
